docs/features/runs/3_faq.md

```diff
@@ -23,7 +23,7 @@ pip freeze > requirements.txt
 It's as easy as running `grid artifacts my-run-name`! This will download all artifacts fromthe run into a new directory called `grid_artifacts`.
 
 ### From the UI
-https://user-images.githubusercontent.com/47154698/146597173-30a6f5af-4ecc-4958-866a-95ddb1ba70e0.mp4
+<Video src="https://user-images.githubusercontent.com/47154698/146597173-30a6f5af-4ecc-4958-866a-95ddb1ba70e0.mp4" type="video/mp4"/> 
```

Comment:
@pritamsoni-hsr this doesn't seem to work when i do `make serve-docs` the video still doesn't show up.
any idea what the best way to add videos is? 
i also tried uploading the video to /images/runs/downloading-artifacts.mp4 then using ![](/path/to/video) but that doesn't work either

Reply:
oh it is working